pkg/apis/core/types.go

```diff
@@ -3590,6 +3590,8 @@ const (
 	ResourceDefaultNamespacePrefix = "kubernetes.io/"
 	// Name prefix for huge page resources (alpha).
 	ResourceHugePagesPrefix = "hugepages-"
+	// Name prefix for storage resource limits
+	ResourceStorageLimitPrefix = "storage-attach-limits-"
```

Comment:
I apologize if this is bikeshedding on the name, but this still reads oddly compared to other resources.

thinking of a node declaring it has a plural count of these resources, and storageclass/pv objects eventually declaring they consume one or more of these resources, this is how this would read:

* node: "capacity: {storage-attach-limits-aws-ebs: 39}"
* pv: "resources: {storage-attach-limits-aws-ebs: 1}"

putting "limits" in the name of the resource is odd... limits are inherent in it being a counted resource. would "attachable-volumes-..." be a better prefix?

* node: "capacity: {attachable-volumes-aws-ebs: 39}"
* pv: "resources: {attachable-volumes-aws-ebs: 1}"

@saad-ali @thockin, thoughts?

Reply:
yeah. naming is hard. I can change it to `attachable-volumes-`  prefix

Reply:
fixed.